qiskit_ibm_runtime/__init__.py

```diff
@@ -162,6 +162,8 @@ def result_callback(job_id, result):
 
 import logging
 
+from qiskit_ibm_provider.utils.version_check import pypi_version_check, update_warning
+
```

Comment:
Since there are plans to deprecate `qiskit-ibm-provider`, I think this file should live in this repo